generate/templates/couchbase-edge-server/Dockerfile.template

```diff
@@ -0,0 +1,46 @@
+FROM {{ .DOCKER_BASE_IMAGE }}
+
+LABEL maintainer="docker@couchbase.com"
+
+# Some ubuntu 22.04 and newer ubuntu have "ubuntu" as the default user, 1000:1000
```

Comment:
Nit: this change was actually made after Ubuntu 22.04. Looks like it was introduced in 23.04.

Otherwise, I think the Dockerfile and generator changes look good.

Reply:
done.